src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -131,6 +131,15 @@ class ReportActionItemMessageEdit extends React.Component {
         Report.saveReportActionDraft(this.props.reportID, this.props.action.reportActionID, '');
         toggleReportActionComposeView(true, this.props.isSmallScreenWidth);
         ReportActionComposeFocusManager.focus();
+
+        // The listener below scrolls to the last comment once the keyboard is hidden to make sure the comment is not hidden and
+        // does not go outside the viewable area in the scroll view. This only applies to the last/latest comment in the report.
```

Comment:
The comment could be improved any thoughts @priyeshshah11 or @dangrous?

Reply:
Does adding this line make it any better? If not, can you please point out what's unclear in the current comment?
```
// This is done because the height of this component changes, which causes it's position to be changed in the scroll view.
```

Reply:
@Santhosh-Sellavel Does adding this line make it any better? If not, can you please point out what's unclear in the current comment?
```
// This is done because the height of this component changes, which causes it's position to be changed in the scroll view.
```

Reply:
A comment is not user-friendly, i.e it's technical.

Reply:
@Santhosh-Sellavel Updated the comment to be technical

Reply:
Sorry, I meant the comment should be user-friendly, and should not be technical i.e not explaining the code.

Reply:
Oh ok, I thought you meant "comments should be technical, not user friendly" which confused me but I think my previous comment was clear enough but I guess I'll reword it

Reply:
@Santhosh-Sellavel simplified it, I think it should be clear enough now.